src/context.rs

```diff
@@ -24,13 +25,8 @@ pub struct Context<'a> {
     /// The map of arguments that were passed when starship was called.
     pub arguments: ArgMatches<'a>,
 
-    /// If `current_dir` is a git repository or is contained within one,
```

Comment:
Perhaps this is a separate PR, but should we use a similar pattern for `dir_files`?

Reply:
I think it would be a pretty big change, so we should do it in a separate PR.

Reply:
Unless you have plans for it, I might try to tackle this in the next few days.